services/dependabot/dependabot.tester.js

```diff
@@ -9,7 +9,7 @@ t.create('semver stability (valid)')
     label: 'semver stability',
     message: isIntegerPercentage,
     link: [
-      'https://dependabot.com/compatibility-score.html?package-manager=bundler&amp;dependency-name=puma&amp;version-scheme=semver',
+      'https://dependabot.com/compatibility-score.html?package-manager=bundler&dependency-name=puma&version-scheme=semver',
```

Comment:
I'd argue the change I made fixed an encoding bug. There is not a good reason for these ampersands to be HTML-encoded in the JSON response.